### PR TITLE
Make list interactive

### PR DESCRIPTION
**Pull request by Michlit7**

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -11,14 +13,24 @@ const MAPLIST = LOCATIONS.map((location, index) => {
 });
 
 export default function Home() {
+  const [currentItemIndex, setCurrentItemIndex] = useState(0);
+
+  function increment() {
```
Comment (demondobf) [inline src/app/page.tsx]: Can we make this name more meaningful?

Comment (Michlit7) [reply src/app/page.tsx]: Yes sir! Would `incrementActive` be more precise?

Comment (demondobf) [reply src/app/page.tsx]: I was thinking about `incrementItemIndex`. Wdyt?

Comment (Michlit7) [reply src/app/page.tsx]: Lovin it!

**Pull request merged**